packages/jasmine/src/jasmine_runner.js

```diff
@@ -151,7 +151,6 @@ function main(args) {
       process.exit(exitCode);
     }
 
-    
```

Comment:
I think code formatting is getting me here. Sorry. I can revert this line if you care.